### DISCUSSION TITLE
Script effetto luci polizia

### DISCUSSION THREAD

**Discussion by texindustries** [indomus/forum#236] [HUB: Home Assistant]
Ciao ragazzi
ho appena integrato magic home, prodotto economico come striscia rgb ma funziona abbastanza bene. Volevo ora fare un piccolo script per fare un effetto sirena della polizia, dove mi alterna luce rossa e luce blu.
Mi pare che negli script non c'è tanta possibilità di fare cicli, ma c'è un modo per farla l'alternanza tra rosso e blu che si ripete all'infinito (poi magari un trigger esterno in automazione interrompe l'esecuzione dello script)?
Grazie mille

**Comment by snakuzzo** [comment]
Credo che gli effetti possibili dipendano dal tipo di device che hai.
Le entità di tipo light prevedono una option ad hoc. Guarda [QUI](https://www.home-assistant.io/integrations/light/)

Dai anche un'occhiata [QUI](https://community.home-assistant.io/t/light-effects-option/21052)

Poi puoi provare anche [QUESTO ESEMPIO](https://community.home-assistant.io/t/custom-light-effects/222020/3) per qualcosa di custom

**Reply by texindustries** [reply]
Grazie Diego... mi aperto un mondo nelle automazioni con il repeat until :) ci darò un occhiata stasera!
